### DISCUSSION TITLE
Can I use the http method QUERY?

### DISCUSSION THREAD

**Discussion by hrstoyanov** [bigskysoftware/htmx#1684] [General]
...even though it is still not approved as a rfc?

https://httpwg.org/http-extensions/draft-ietf-httpbis-safe-method-w-body.html

**Comment by Telroshan** [comment]
Probably too early as it isn't even mentioned in the [MDN doc](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods) yet, nor on [caniuse](https://caniuse.com/?search=http%20method%20query)
Though, it shouldn't be that complicated to implement since it would just require to define a `hx-query` attribute.
I would say, feel free to open a PR and work on it, but I wouldn't expect it to be merged until the QUERY method gets approved and becomes a standard
